cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -111,6 +111,7 @@ message SearchSeismicStoresRequest {
     bool include_headers = 5; // if true, include text and binary headers in the response
     bool include_coverage = 4; // Deprecated. Use `coverage` instead.
     CoverageSpec coverage = 7; // If specified, include coverage
+    bool volumedef_as_msgpack = 8; // If true and include_volume_defs is true, outputs as msgpack
```

Comment:
Make the tag number high? Say, 500000000 or something? Then it'll stand out as a bit of an oddball and maybe remind us to remove it again...
https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers

Reply:
Will do